com.unity.robotics.urdf-importer/Runtime/Extensions/MaterialExtensions.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using UnityEngine;
+using UnityEngine.Rendering;
+
+namespace Unity.Robotics
+{
+    public static class MaterialExtensions
+    {
+        private static string[] standardShaders = { "Standard" };//, "UI/Default" };
+        private static string[] hdrpShaders = { "HDRP/Lit" };//, "UI/Default" };
+        public static Material CreateBasicMaterial()
+        {
+            try
+            {
+                string[] shadersToTry = IsHDRP() ?  hdrpShaders : standardShaders;
+                foreach (var shaderName in shadersToTry)
+                {
+                    Shader shader = Shader.Find(shaderName);
+                    if (shader != null)
+                    {
+                        //Debug.Log("Found shader: " + shaderName);
+                        var material = new Material(shader);
+                        material.SetFloat("_Metallic", 0.75f);
+                        material.SetFloat("_Glossiness", 0.75f);
+                        return material;
+                    }
+                }
+            }
+            catch (Exception ex)
+            {
+                Debug.LogAssertion(ex.ToString());
+            }
+            return null;
```

Comment:
Just curious what will happen if the Material is `null`? It seems to me the code that uses this method will throw an exception. Would it be safer to return a default Material?

Reply:
A null material will show a Pink. The LogAssertion is so that we get the stack trace on where things went wrong, but it won't throw an exception. The problem with having a default material is that we still need to create it in run-time and that requires assigning it a shader, which in return can again result in a null material if the shader is not found.